Sources/MongoSwift/MongoClient.swift

```diff
@@ -272,14 +292,8 @@ public class SyncMongoClient {
      * - Parameters:
      *   - pointer: the `mongoc_client_t` to store and use internally
      */
-    public init(stealing pointer: OpaquePointer) {
-        self.connectionPool = ConnectionPool(stealing: pointer)
-        self.encoder = BSONEncoder()
-        self.decoder = BSONDecoder()
-        self.readConcern = nil
-        self.readPreference = ReadPreference()
-        self.writeConcern = nil
-        self.notificationCenter = NotificationCenter.default
+    override public init(stealing pointer: OpaquePointer) {
```

Comment:
I kind of wonder if we should just remove this from the public API altogether. we definitely don't need to add a corresponding async init but I'm not sure we even need this one (or "single" mode in the pool) anymore.

Reply:
I think given the current mobile situation, this can probably be removed.